crates/pop-parachains/src/templates.rs

```diff
@@ -108,8 +108,13 @@ impl Template {
 		self.get_str("Provider") == Some(provider.to_string().as_str())
 	}
 
-	pub fn repository_url(&self) -> Result<&str, Error> {
-		self.get_str("Repository").ok_or(Error::RepositoryMissing)
+	pub fn repository_url(&self) -> Result<String, Error> {
+		Ok(["https://github.com/", self.get_str("Repository").ok_or(Error::RepositoryMissing)?]
```

Comment:
Still dont think this is necessary when you can just define `Repository = "https://github.com/paritytech/frontier-parachain-template"` on the enum. There is a minor additional size but it simplifies the code, reduces the concat at runtime and also allows any non-GH repo url to be defined.

Reply:
Aaah, I now see it is probably due to the ssh url

Reply:
Still, your points are very valid. Does make sense to have something like this to avoid the concat:
```rs
#[default]
	#[strum(
		serialize = "base",
		message = "Standard",
		detailed_message = "A standard parachain",
		props(
			Provider = "Pop",
			Repository = "https://github.com/r0gue-io/base-parachain",
			SSHRepository = "git@github.com:r0gue-io/base-parachain.git"
		)
	)]
	Base,
```

Reply:
Looking at the usage, the string is being parsed into a `Url` type. That can easily be used to create a new url in the case of ssh - something like `format!("git@{}:{}", url.host_str(), url.path())`

https://docs.rs/url/latest/url/

Reply:
This is a nice approach